FlightsFile.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Text;
+
+namespace AirportTicketBooking.CSVFiles
+{
+    public static class FlightsFile
```

Comment:
rename to FlightService 
and we have the same concern here, we have to separate the file I/O to another class, and fill in a list of objects to use here, this will reduce the times when we load the file and open file streams